docs: note that Lens auto-selects Line for date fields (#6061) (#6081)
elastic/docs-content: explore-analyze/visualize/charts/line-charts.md

:::::

:::::{step} Set the visualization to Line

> New visualizations default to **Bar**. Using the visualization type dropdown, select **Line**.

Author: ```suggestion
Make sure that the visualization type is set to **Line**.
```

> {applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` When you start by dragging a date or time field (such as `@timestamp`) onto an empty workspace, **Line** is automatically selected.

Reviewer: ```suggestion
{applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` When you drag a date or time field (such as `@timestamp`) onto an empty workspace, **Line** is automatically selected.
```

> {applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` You can skip this step when using a data view if you start by dragging a date or time field (such as `@timestamp`) onto an empty workspace: Lens automatically selects **Line** in that case. The manual switch is still required for [ES|QL](../esorql.md) queries or when the first field you drag is not a date or time field.

Author: ```suggestion
{applies_to}`stack: ga 9.4` {applies_to}`serverless: ga` When you start by dragging a date or time field (such as `@timestamp`) onto an empty workspace, **Line** is automatically selected.
```
